src/main/java/com/teamrocket/tms/services/user/UserServiceImpl.java

```diff
@@ -31,14 +33,17 @@ public class UserServiceImpl implements UserService {
     private final ProjectService projectService;
 
     private final TeamService teamService;
+    private final ProjectRepository projectRepository;
```

Comment:
Maybe instead of injecting ProjectRepository we can make use of the projectService that has already the implementation for project deletion?
Usually we don't want for services to have direct access to other entities repository :)